Source/WebKit/WebProcess/GPU/graphics/WebGPU/RemoteQueueProxy.cpp

```diff
@@ -81,7 +81,7 @@ void RemoteQueueProxy::writeBuffer(
     if (!convertedBuffer)
         return;
 
-    auto sendResult = send(Messages::RemoteQueue::WriteBuffer(convertedBuffer, bufferOffset, Vector<uint8_t>(static_cast<const uint8_t*>(source) + dataOffset, size.value_or(byteLength - dataOffset))));
```

Comment:
I don't think we can ignore the `size` member, can you please correct before merging?

Reply:
It looks like I did inadvertently drop `size` in the new code. I'll fix that. Thanks for catching.